app/views/booths/_form.html.haml

```diff
@@ -12,6 +12,8 @@
           hint: 'e.g. employee, comunity manager, etc'
         = f.input :website_url
         = responsibles_selector_input f
+        = f.input :invite_responsible,
+          hint: 'This field is to invite unregistered users using their email to become stand organizers.'
```

Comment:
I don't understand how this is going to be helpful though :/ 

People seeing this interface know about Booths, now here you mention the word stand. Isn't that confusing?

Reply:
@differentreality true. Should I add booth responsible in brackets at the end?

Reply:
No! We need to come up with an approach for this in general. Not just this little hint here.

Reply:
If this is for fosdem only, then can't we just rename booth to stand at every place?

Reply:
This, as in the GSoC19 branch? No, this is not necessarily just for FOSDEM. But it will definitely be if we introduce changes that make it impossible to merge it back to master!

Reply:
@differentreality what about creating locales for booth? We can create a "naming_convention.yml" in "config/locales" and allow users to change to booth or stand.

Reply:
That sounds like a plan! 

So here we should keep, for now, something that makes sense to the rest of the wording being used, ie Booths. 

And a new PR should take care of the Stands wording wherever needed.

Reply:
right.